src/components/pages/Partnerships/Partnerships.tsx

```diff
@@ -47,7 +49,21 @@ export default function Partnerships() {
                   })}
                 </h3>
               </div>
-              <div style={{ height: "70vh" }} className="row">
+              { isMobileDeviceOrTablet ? 
```

Comment:
Nit: this looks a bit messy, my preference would be to return both the mobile/desktop views here from a function so that the jsx ternary looks cleaner here

Reply:
you're right, just wanted to fix the view on mobile so that it wasn't completely broken

Reply:
I'll add a TODO